qiita_pet/handlers/api_proxy/studies.py

```diff
@@ -0,0 +1,71 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from __future__ import division
+
+from qiita_db.study import Study
+from qiita_pet.handlers.api_proxy.util import check_access
+
+
+def data_types_get_req():
+    """Returns data types available in the system
+
+    Returns
+    -------
+    list of str
+        Data types available on the system
+    """
+    return {'status': 'success',
+            'message': '',
+            'data_types': Study.all_data_types()
```

Comment:
A study tells you what data types it has, so it makes sense that the uninstantiated study can tell you all data types available. Also, as you have stated, this has already been reviewed and merged previously, with no comment before.  If you didn't agree with that, you should raise up your concern then.